moveit_planners/trapezoidal/CMakeLists.txt

```diff
@@ -0,0 +1,547 @@
+            src/joint_limits_aggregator.cpp  # do we need joint limits and cartesian_limit here?
+            src/joint_limits_container.cpp
+            src/limits_container.cpp
+            src/cartesian_limit.cpp
+            src/cartesian_limits_aggregator.cpp
+            )
+target_link_libraries(sequence_capability
+                      ${catkin_LIBRARIES}) # DO NOT LINK ${PROJECT_NAME} here!
+add_dependencies(sequence_capability
+           ${catkin_EXPORTED_TARGETS})
+
+#############
+## Install ##
+#############
+install(FILES
+       plugins/sequence_capability_plugin_description.xml
+       plugins/trapezoidal_command_planner_plugin_description.xml
+       plugins/planning_context_plugin_description.xml
+       DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/plugins
+       )
+
+## Mark libraries for installation
+install(TARGETS
+   trapezoidal_command_planner
+   planning_context_loader_ptp
+   planning_context_loader_lin
+   planning_context_loader_circ
+   command_list_manager
+   sequence_capability
+#   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
+## Mark cpp header files for installation
+#install(DIRECTORY include/${PROJECT_NAME}/
+#  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
+#  FILES_MATCHING PATTERN "*.h"
+#  PATTERN ".svn" EXCLUDE
+#)
+
+#############
+## Testing ##
+#############
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+  find_package(pilz_testutils REQUIRED)
+
+  include_directories(
+    ${pilz_testutils_INCLUDE_DIRS}
+  )
+
+  if(ENABLE_COVERAGE_TESTING)
+    include(CodeCoverage)
+    APPEND_COVERAGE_COMPILER_FLAGS()
+  endif()
+
+  #########################
+  ####Integration Tests####
+  #########################
+
+  set(${PROJECT_NAME}_INTEGRATIONTEST_LIBRARIES
+    sequence_capability
+    trapezoidal_command_planner
+    planning_context_loader_ptp
+    planning_context_loader_lin
+    planning_context_loader_circ
+  )
+
+  # Planning Integration tests
+  add_rostest_gmock(integrationtest_sequence_action_capability
+    test/integrationtest_sequence_action_capability.test
+    test/integrationtest_sequence_action_capability.cpp
+  )
+
+  target_link_libraries(integrationtest_sequence_action_capability
+    ${catkin_LIBRARIES}
+    ${pilz_testutils_LIBRARIES}
+    ${${PROJECT_NAME}_INTEGRATIONTEST_LIBRARIES}
+  )
+
+  add_rostest(test/integrationtest_sequence_action_capability_with_gripper.test
+    DEPENDENCIES integrationtest_sequence_action_capability
+  )
+
+  # Tests with other robots only for kinetic (for now)
+
+  #add_rostest(test/integrationtest_sequence_action_capability_abb_irb2400.test
+  #  DEPENDENCIES integrationtest_sequence_action_capability
+  #)
+
+  #add_rostest(test/integrationtest_sequence_action_capability_frankaemika_panda.test
+  #  DEPENDENCIES integrationtest_sequence_action_capability
+  #)
```

Comment:
Why are Franka tests disabled? we support those in moveit_resources for tests:
https://github.com/ros-planning/moveit_resources/tree/master/panda_moveit_config

Reply:
The issue https://github.com/PilzDE/pilz_industrial_motion/issues/176 doesn't tell the problem or link a failed build - we'd have to try enabling them again...

Reply:
I enabled them again. Closes https://github.com/PilzDE/pilz_industrial_motion/issues/176